modules/ding_eresource/ding_eresource.pages_default.inc

```diff
@@ -102,6 +102,8 @@ function ding_eresource_default_page_manager_handlers() {
   $display->cache = array();
   $display->title = '';
   $display->uuid = 'd82976d5-f78c-46b3-8d9a-c2ba458d539c';
+  $display->storage_type = 'page_manager';
```

Comment:
How is this related to the case?

Reply:
This is required by panels module in order to fix some access permissions issues, introduced in version 7.x-3.6, when on pane editing administrators had "access denied" error.